src/option_parser.c

```diff
@@ -141,6 +141,27 @@ bool string_parse_mouse_action(const char *s, enum mouse_action *ret)
         return false;
 }
 
+bool string_parse_mouse_action_list(char **s, enum mouse_action **ret)
+{
+        ASSERT_OR_RET(s, false);
+        ASSERT_OR_RET(ret, false);
+
+        int len = 0;
+        while (s[len])
+                len++;
+
+        *ret = g_malloc((len + 1) * sizeof(enum mouse_action));
```

Comment:
Please use `g_n_malloc()` it avoids buffer overflows.

Reply:
Sorry, it's called `g_malloc_n()`. See here: https://developer.gnome.org/glib/stable/glib-Memory-Allocation.html#g-malloc-n

Reply:
Thanks, will do.